lib/measurable/interfaces.rb

```diff
@@ -0,0 +1,52 @@
+module Measurable
+  module Interface
+    class DistanceByName
+      def initialize(name)
+        @name = name
+      end
+
+      def distance(a, b)
+        Measurable.send(@name, a, b)
+      end
+    end
+    def getObject(measure)
```

Comment:
In Ruby, we generally don't use camelCase in method names; use `get_object` instead. Also, `get_object` seems like a bad method name... how about `object_for(measure)`? Or `object_by(name)`?